README.md

```diff
@@ -23,7 +27,7 @@ Neovim is supported.
 
 ## Installation
 
-With [vivify](https://github.com/jannis-baum/vivify) installed in your PATH, you
+With [Vivify](https://github.com/jannis-baum/vivify) installed in your PATH, you
```

Comment:
```suggestion
With [Vivify](https://github.com/jannis-baum/Vivify) installed in your PATH, you
```

Capitalize in the URL too? (extreme nitpick)

Reply:
Good you spotted that! fixed in https://github.com/jannis-baum/vivify.vim/pull/29/commits/3d36a83485965d04cefa129c594098d990b3ebc6